code/mission/missioncampaign.h

```diff
@@ -30,6 +30,8 @@ struct sexp_variable;
 #define CAMPAIGN_ERROR_SAVEFILE			-4
 #define CAMPAIGN_ERROR_IGNORED			-5
 
+inline bool campaign_reportable_error(int val) { return val != 0 && val != CAMPAIGN_ERROR_MISSING && val != CAMPAIGN_ERROR_IGNORED; }
```

Comment:
Unless I've missed it, this doesn't appear to be used outside of `missioncampaign.cpp`. If that's the case it would be best to just move it there so that it's not global.

Reply:
True, it's not used outside the .cpp file, but since the #defines are in the header, I thought this function should be too.  It represents an essential property of the different error types.

Reply:
And when you're running on a craptop like me, compilation time matters a lot.

Reply:
Ok, done.